src/event.rs

```diff
@@ -640,14 +640,14 @@ pub enum MouseScrollDelta {
     ///
     /// Positive values indicate movement forward
     /// (away from the user) or rightwards.
```

Comment:
```suggestion
    /// Positive values indicate upward and rightward movement.
```

Reply:
This has nothing to do with this PR, so I don't see why I would touch any of that?

I can't say if the original statement is correct, since it does not really indicate any concrete direction ("the user" is in front of the screen, so I doubt it moves away from him).

Reply:
Why not fix the comment while we're here? It'll never get fixed at this rate.

Reply:
Because I do not think this comment makes much sense anyways and I do not plan to argue about it in this PR. If you are more invested in bikeshedding about it, there's nothing wrong with opening a separate PR?

I'm fine with changing it in this PR if there's a direct suggestion from winit's maintainers, but I do not wish to invest my time to do anything about this, or have this PR unnecessarily held back because of it.

Reply:
@chrisduerr that's fair. I should mark my comments with (no-block).